proposed/event-dispatcher.md

```diff
@@ -11,75 +11,77 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 ## Goal
 
-Having common interfaces for dispatching and handling events, allows developers to create libraries that can interact with many frameworks and other libraries in a common fashion.
+Having common interfaces for dispatching and handling events allows developers to create libraries that can interact with many frameworks and other libraries in a common fashion.
 
 Some examples:
 
 * A security framework that will prevent saving/accessing data when a user doesn't have permission.
 * A common full page caching system.
-* Libraries that extent other specific libraries, regardless of what framework they are both integrated into.
+* Libraries that extend other libraries, regardless of what framework they are both integrated into.
 * A logging package to track all actions taken within the application
 
 ## Definitions
 
 * **Event** - An Event is a message produced by an *Emitter*.  It may be any arbitrary PHP object.
 * **Listener** - A Listener is any PHP callable that expects to be passed an Event.  Zero or more Listeners may be passed the same Event.  A Listener MAY enqueue some other asynchronous behavior if it so chooses.
-* **Emitter** - An Emitter is any arbitrary code that wishes to send an Event.  This is also known as the "calling code".  It is not represented by any particular data structure but refers to the use case.
+* **Emitter** - An Emitter is any arbitrary code that wishes to dispatch an Event.  This is also known as the "calling code".  It is not represented by any particular data structure but refers to the use case.
```

Comment:
I did a `s/send/dispatch/` here, to have affinity with the `Dispatcher` term we bring up next.